components/brave_wallet/common/zcash_utils.cc

```diff
@@ -29,11 +102,11 @@ DecodedZCashAddress::DecodedZCashAddress(DecodedZCashAddress&& other) = default;
 DecodedZCashAddress& DecodedZCashAddress::operator=(
     DecodedZCashAddress&& other) = default;
 
-bool IsValidZCashAddress(const std::string& address) {
-  return true;
+bool IsUnifiedAddress(const std::string& address) {
+  return address.starts_with("u1") || address.starts_with("utest1");
```

Comment:
Is it worth doing a length check here too?

Reply:
For unified addresses length is not fixed since it could contain multiply sub addresses, we need to parse address to verify

Reply:
cool, thanks for pointing that out. This seems fine to me otherwise